site/app/controllers/GlobalController.php

```diff
@@ -211,43 +211,14 @@ public function header() {
             }
 
             if ($this->core->getUser()->accessGrading()) {
-                $images_course_path = $this->core->getConfig()->getCoursePath();
-                // FIXME: this code is duplicated in ImagesController.php
-                $images_path = FileUtils::joinPaths($images_course_path, "uploads/student_images");
-                $common_images_path_1 = FileUtils::joinPaths("/var/local/submitty/student_images");
-                $term = explode('/', $this->core->getConfig()->getCoursePath());
-                $term = $term[count($term) - 2];
-                $common_images_path_2 = FileUtils::joinPaths("/var/local/submitty/student_images", $term);
-                // FIXME: consider searching through the common location for matches to my students
-                // (but this would be expensive)
-                $any_images_files = array_merge(
-                    FileUtils::getAllFiles($images_path, [], true),
-                    FileUtils::getAllFiles($common_images_path_1, [], true),
-                    FileUtils::getAllFiles($common_images_path_2, [], true)
-                );
-                if ($this->core->getUser()->accessAdmin() && count($any_images_files) === 0) {
-                    $at_least_one_grader_link = true;
-                    $sidebar_buttons[] = new Button($this->core, [
-                        "href" => $this->core->buildCourseUrl(['student_photos']),
-                        "title" => "Student Photos",
-                        "class" => "nav-row",
-                        "id" => "nav-sidebar-photos",
-                        "icon" => "fa-id-card"
-                    ]);
-                }
-                elseif (count($any_images_files) !== 0 && $this->core->getUser()->accessGrading()) {
-                    $sections = $this->core->getUser()->getGradingRegistrationSections();
-                    if (!empty($sections) || $this->core->getUser()->getGroup() !== User::GROUP_LIMITED_ACCESS_GRADER) {
-                        $at_least_one_grader_link = true;
-                        $sidebar_buttons[] = new Button($this->core, [
-                            "href" => $this->core->buildCourseUrl(['student_photos']),
-                            "title" => "Student Photos",
-                            "class" => "nav-row",
-                            "id" => "nav-sidebar-photos",
-                            "icon" => "fa-id-card"
-                        ]);
-                    }
-                }
+                $at_least_one_grader_link = true;
```

Comment:
this (and all usages of it) can be removed. There's never a time where `$this->core->getUser()->accessGrading()` will be `true` and `$at_least_one_grader_link` will be `false`.

Reply:
Done